pkg/platform/runtime/setup/setup.go

```diff
@@ -290,11 +281,7 @@ func (s *Setup) Update() error {
 		return errs.Wrap(err, "Could not save recipe file.")
 	}
 
-	if err := s.store.MarkInstallationComplete(s.target.CommitUUID()); err != nil {
-		return errs.Wrap(err, "Could not mark install as complete.")
-	}
-
-	return nil
+	return s.completeSetup(buildResult.OrderedArtifacts())
```

Comment:
Rather than move this logic into a new `completeSetup()` function couldn't this instead be placed in the `Update()` function after it calls the `if/else` for offline vs online installs. Reduces the number of layers to jump through when grokking code and more clearly conveys the desired behaviour of the code.